account_product_fiscal_classification/__init__.py

```diff
@@ -1,49 +1 @@
-# -*- coding: utf-8 -*-
-# Copyright (C) 2014-Today GRAP (http://www.grap.coop)
-# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
-import logging
-
-from odoo import api, SUPERUSER_ID
-
 from . import models
-
-_logger = logging.getLogger(__name__)
-
-
-def create_fiscal_classification_from_product_template(cr, registry):
```

Comment:
I don't understand why you delete this function ? It is usefull to populate database with existing products.